modules/retail/pages/retail-admin-network.adoc

```diff
@@ -0,0 +1,19 @@
+[[retail-admin-network]]
+= Network Administration
+
+If you are intending to set up either an external or a shared network architecture, you need to ensure that the server providing DHCP services has PXE support enabled.
+
+For example:
+
+----
+next-server: <branch server IP>
+if option arch = 00:07 {
+     filename "boot/shim.efi";
+} else {
+     filename "boot/pxelinux.0";
+}
+----
+
+Your DNS servers can then resolve ``salt`` and ``ftp`` hostnames as CNAMEs to the correct branch server hostname.
```

Comment:
Not `can`, but must.
Customer need to configure their DNS servers in a way that at minimum `salt` is CNAME and resolve to the correct branch server FQDN. `ftp` should resolve to the branch as well, but can be regular A record (resolving to IP), no need for CNAME.
This is the hardest part of all of the retail to configure correctly.

If this is impossible or hard for customer to guarantee these DNS entries, there are workarounds:
* for `salt` instead of using CNAME, pass `MASTER=<branch_server_fqdn>` on kernel command line when terminal is booting up. PXE formula can be used to configure this value (see https://documentation.suse.com/external-tree/en-us/suma/4.1/suse-manager/salt/formula-pxe.html )
* for `ftp` see  https://documentation.suse.com/external-tree/en-us/suma/4.1/suse-manager/retail/retail-deploy-terminals.html#_customize_the_terminal_image_download_process about using `saltboot_download_server`

Reply:
OK, I've added this in now.